jujugui/static/gui/src/app/components/service-overview/test-service-overview.js

```diff
@@ -35,8 +35,17 @@ describe('ServiceOverview', function() {
   });
 
   it('generates a list of actions', function() {
```

Comment:
Could you add a few more tests here making sure that it's actually rendering correctly based on different sets of unit data passed in.